### ISSUE TITLE
Can the EcsCluster provide for adding custom tags to the auto scaling group?

### ISSUE THREAD

**Issue by gillfimj** [sigdba/sig-shared-sceptre#11]
Some clients may have a need to add custom tags to any EC2 instances that get created by the ASG.  While this can be added manually via the console, it would be nice if this was configurable in the yaml file.

**Comment by gillfimj** [comment]
Forgot to add, and subsequently add those same tags to any volumes that the EC2 instance creates.

**Comment by gabetocci** [comment]
yep. You can add tags to a stack via the cli when you create/update the stack via --tag argument. If you want to define it in the template, use a nested stack https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-stack.html and then use GetAtt https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-getatt.html in the EC2 volume template
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-ebs-volume.html

**Comment by gillfimj** [comment]
Looks like the main.py already has the provision to add one tag:

Tags=ASTags(Name=Sub("ecs-node-${AWS::StackName}")),

Probably just need to expand this to allow additional tags.

**Comment by dboitnot** [comment]
Added in release 30.
